hw/xbox/nv2a/nv2a_vsh.c

```diff
@@ -358,7 +358,8 @@ static QString* decode_opcode_input(const uint32_t *shader_token,
     case PARAM_C:
         reg_num = convert_c_register(vsh_get_field(shader_token, FLD_CONST));
         if (vsh_get_field(shader_token, FLD_A0X) > 0) {
-            //FIXME: does this really require the "correction" doe in convert_c_register?!
+            // FIXME: does this really require the "correction" doe in
+            // convert_c_register?!
```

Comment:
If we break this anyway, we could also just use C89 style comment here.
There's other `/* */ ` comments anyway.